appshell/client_handler.h

```diff
@@ -16,7 +16,13 @@
 
 #include <algorithm> 
 
-
+#if defined(OS_LINUX)
+// The Linux client uses GTK instead of the underlying platform type (X11).
```

Comment:
The change to define `ClientWindowHandle` to be `GtkWidget*` is pulled straight from the cefclient sample app from CEF https://code.google.com/p/chromiumembedded/source/browse/branches/2171/cef3/tests/cefclient/client_handler.h#19.